src/client/components/pages/entity-revisions.js

```diff
@@ -54,6 +70,12 @@ class EntityRevisions extends React.Component {
 		);
 	}
 
+/**
+* Renders the data related to Revision such as 'author' and 'date'.
+* It also displays 'Revision Notes' column.
```

Comment:
It actually displays the first revision note, which should be a summary of the changes made in the revision